index.html

```diff
@@ -5,8 +5,55 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
 </head>
-<body>
-    
+
+<body class="foggy">
```

Comment:
Your HTML page looks good. The use of your section tags and appropriate tags within the sections makes your code readable.